.jscsrc

```diff
@@ -9,6 +9,9 @@
 	"disallowMultipleLineStrings": true,
 	"disallowNewlineBeforeBlockStatements": true,
 	"disallowOperatorBeforeLineBreak": [
+		"?",
+		"||",
+		":",
```

Comment:
I think this will break switch case statements:
```
switch(expression) {
    case n:
        code block
        break;
    case n:
        code block
        break;
    default:
        default code block
}
```